docs/pages/architecture/overview.mdx

```diff
@@ -234,16 +227,17 @@ valid for 12 hours by default, unless you specify another interval via the
 
 You can learn more about `tsh` in the [User Manual](../server-access/guides/tsh.mdx).
 
-### TCTL
+### `tctl`
 
-`tctl` is used to administer a Teleport cluster. It connects to the Auth
-server listening on `127.0.0.1` and allows a cluster administrator to manage
-nodes and users in the cluster.
+`tctl` is used to administer a Teleport cluster. It connects to the Teleport
+Auth Service and allows an administrator to manage Nodes, users, and other
+resources in the cluster.
 
-`tctl` is also a tool that can be used to modify the dynamic configuration of
-the cluster, like creating new user roles or connecting trusted clusters.
+When running `tctl`, a user must authenticate to a Teleport cluster. `tctl` can
```

Comment:
I would tweak the wording here a bit. When running on an auth server directly, tctl will use the identity of the auth server itself when talking to its API, you don't need any extra authentication. When running on a client host, user needs to log into the cluster with tsh and tctl will use the user's identity created by tsh login.